### PR TITLE
Buttons animation

### PR DESCRIPTION
**Pull request by lkkolosowski**

### CODE REVIEW

File: src/common/Header/Navigation/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { StyledNavigation, Item, StyledNavLink } from "./styled";
 const Navigation = () => (
     <StyledNavigation>
       <Item>
-        <StyledNavLink to="/movies">MOVIES</StyledNavLink>
+        <StyledNavLink movie="true" to="/movies">MOVIES</StyledNavLink>
```
Comment (Albrecht-Albi) [inline src/common/Header/Navigation/index.js]: Good change

File: src/common/Header/Navigation/styled.js [hunk 2]
```diff
@@ -11,26 +11,64 @@ export const StyledNavigation = styled.ul`
...
+    border-bottom: 1px solid ${({ theme }) => theme.color.white};
+    bottom: 0;
+    width: calc(100% - (var(--margin) * 2));
+    margin: 0 var(--margin);
+    transform: rotateY(90deg);
+    transition: transform 0.15s;
+  }
+
+  &:hover:after {
+    transform: rotateY(0deg);
```
Comment (Albrecht-Albi) [inline src/common/Header/Navigation/styled.js:36]: Looks much better now, great

File: src/common/Header/Navigation/styled.js [hunk 3]
```diff
@@ -11,26 +11,64 @@ export const StyledNavigation = styled.ul`
...
+        position: absolute;
+        inset: 0;
+        border: 1px solid ${({ theme }) => theme.color.white};
+        border-radius: 33px;
+        transform: translateX(calc(100% + 10px));
+      }
+
+      &.active:before {
+        transform: translateX(0%);
+      }
```
Comment (Albrecht-Albi) [inline src/common/Header/Navigation/styled.js:70]: I like very nice layout

**Review by Albrecht-Albi — APPROVED** [review]

File: src/common/Header/Navigation/styled.js (hunk 2 above)
Comment (PawelDzi) [inline src/common/Header/Navigation/styled.js:36]: good job

**Review by PawelDzi — APPROVED** [review]

**Pull request merged**